### PR TITLE
moov_cruze

### PR DESCRIPTION
**Pull request by iiMOOV**
By @iiMoov

### CODE REVIEW

File: config/cars/mods/moov_cruze [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+
+[Material_PhotoelasticGlass]
+Materials =  Material #46
+FilmIOR = 2
+ThicknessMult = 1.2
+Desaturate = 0
+MaskPass = 5
+
+
+[INCLUDE: common/materials_glass.ini]
```
Comment (Marocco2) [inline config/cars/mods/moov_cruze:185]: Duplicated include

File: config/cars/mods/moov_cruze [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+
+FALLBACK_EMISSIVE=25, 12, 3
+LAYOUT=TWO_ON_TOP
+FADE_AT=500000
+FALLBACK_LOWBEAM_COLOR=25, 12, 3
+
+----------------------------------------------------
+[INCLUDE: common/materials_carpaint.ini]
+CarPaintMaterial = Material #CARPAINT,
+DisableDev = 1
```
Comment (Marocco2) [inline config/cars/mods/moov_cruze:120]: (optional) Add `CarPaintVersionAware = 2` and check results. Default value is 1

File: config/cars/mods/moov_cruze [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+Materials =  Material #96
+FilmIOR = 1.4
+ThicknessMult = 0.5
+Desaturate = 0
+MaskPass =0
+
+
+[SHADER_REPLACEMENT_...]
+MATERIALS = Material #46
+DOUBLE_FACE_SHADOW_BIASED = 0.0
```
Comment (Marocco2) [inline config/cars/mods/moov_cruze:198]: Default value is already 0. Also it's a boolean (0 or 1)

**Review by Marocco2 — CHANGES_REQUESTED** [review]
Check comments. Try also clean up empty lines

**Comment by iiMOOV** [comment]
ok my bad i do it

>

**Review by Marocco2 — APPROVED** [review]

**Pull request merged**